exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/ddl/SequencesTests.kt

```diff
@@ -311,6 +318,48 @@ class SequencesTests : DatabaseTestsBase() {
         }
     }
 
+    @Test
+    fun testAutoIncrementColumnAccessWithEntity() {
+        Assume.assumeTrue(TestDB.POSTGRESQL in TestDB.enabledDialects())
+
+        Database.connect(
+            TestDB.POSTGRESQL.connection(),
+            TestDB.POSTGRESQL.driver,
+            TestDB.POSTGRESQL.user,
+            TestDB.POSTGRESQL.pass
+        )
```

Comment:
Minor fyi: Since none of the parameters are being customized, this entire call could be replaced with `TestDB.POSTGRESQL.connect()`.